.vib/harbor-jobservice/goss/harbor-jobservice.yaml

```diff
@@ -0,0 +1,25 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /var/log/jobs:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+command:
+  # Ensure swagger binary doesn't exist
+  check-swagger-binary:
+    exec: ls /opt/bitnami/harbor-jobservice/bin/swagger 2>/dev/null
+    exit-status: 2
```

Comment:
I would move this to a `file` Goss test:
```
file:
  /opt/bitnami/harbor-jobservice/bin/swagger:
    exists: false
```

I think this way we use the in-built features of Goss and a cleaner check, wdyt?